src/Preferences.js

```diff
@@ -65,6 +65,15 @@ define(function (require, exports, module) {
                 description: Strings.PREF_INDENT_DESC
             },
             inContextMenu: true
+        }, {
+            id: "autohide",
+            type: "boolean",
+            value: false,
+            options: {
+                name: Strings.PREF_AUTOHIDE_NAME,
+                description: Strings.PREF_AUTOHIDE_DESC
+            },
+            inContextMenu: false
```

Comment:
I think you can remove this. It isn't present in the other preferences where it is false.

Alternatively add it everywhere. I just like it consistent.